src/scripts/main.js

```diff
@@ -1,3 +1,50 @@
 'use strict';
 
-// write your code here
+const logo = document.getElementById('logo');
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve();
+    reject(new Error('Promise was rejected!'));
+  });
+});
+
+promise1.then(() => {
+  const box1 = document.createElement('div');
+
+  document.body.appendChild(box1);
+  box1.className += 'message';
+  box1.textContent += 'Promise was resolved!';
```

Comment:
You can combine the code for creating the message boxes into a repeatable function to avoid repeating the same code.